src/theme/src/hooks/useTableCellAppearance.js

```diff
@@ -0,0 +1,23 @@
+import { useMemo } from 'react'
+import useStyleConfig from '../../../hooks/use-style-config'
+import getTableCellStyles from '../components/table-cell'
+import useTheme from '../useTheme'
+
+const pseudoSelectors = {
+  _focus:
+    '&[data-isselectable="true"]:focus, &[aria-expanded="true"][aria-haspopup="true"]'
```

Comment:
Curious why we need `&[data-isselectable="true']:focus` instead of `&:focus`?

Reply:
We only want to apply focus styles to a selectable table cell - unsure of what reason, but this is preserved behavior.

Reply:
I wonder if the only cells that can receive focus on the ones that are selectable though...